sdk/src/store.rs

```diff
@@ -1480,6 +1481,11 @@ impl Store {
     ) -> Result<()> {
         // walk the ingredients
         for i in claim.ingredient_assertions() {
+            // allow for zero out ingredient assertions
+            if is_zero(i.assertion().data()) {
```

Comment:
I thought you said we would not see these as assertions. Is there a case where we do?

Reply:
We don't be it is a new required test, so I added it.